src/nccl_ofi_rdma.c

```diff
@@ -4789,11 +4794,11 @@ static int listen(nccl_net_ofi_ep_t *base_ep,
 	goto exit;
 
  error:
-	if (l_comm && ~0 != l_comm->comm_id) {
-		if (0 != nccl_ofi_idpool_free_id(device->comm_idpool, l_comm->comm_id)) {
-			NCCL_OFI_WARN("Error freeing communicator ID %" PRIu32, l_comm->comm_id);
-		}
-	}
+	 if (l_comm && COMM_ID_INVALID != l_comm->comm_id) {
```

Comment:
github says there's leading spaces here that shouldn't be.

Reply:
I took the entire series and rebased with `git clang-format HEAD~1` to format only the lines changed. I thought maybe I missed this line, so I rebased but it did not change anything. I made the change manually to remove the space, but running clang-format against [this config](https://gist.github.com/aws-nslick/df84cb5d739a5e2640a6e2b1fed38eb6) reinserts the space, ie: it wants that space to be there.

I uploaded the file to https://zed0.co.uk/clang-format-configurator/ and pasted in this function, and it reformats the label. I think this is a bug in logic in git-clang-format, where it will not extend the "modified" region to include goto labels, and then it assumes that it's in a region that is indented with one space, and so it is keeping that indentation level and adding to it. Or something. I dunno.

I have removed the leading space from the label and rebased.